pkg/cluster/logs/logs.go

```diff
@@ -87,6 +87,10 @@ func Collect(nodes []nodes.Node, dir string) error {
 					filepath.Join(name, "inspect.json"),
 				),
 				// grab all of the node logs
+				execToPathFn(
+					exec.Command("docker", "logs", name),
```

Comment:
should we add `-t` to the command to have the timestamps?
I find them very useful

Reply:
https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/sigs.k8s.io_kind/480/pull-kind-conformance-parallel-1-14/1124528321279823874/artifacts/logs/kind-control-plane/serial.log

probably not so useful. I do think we might want to tweak the init output though.

Reply:
also definitely need to refactor this out to pkg/container, have a bigger PR to refactor this code later.